ERCS/erc-7734.md

```diff
@@ -0,0 +1,192 @@
+---
+eip: 7734
+title: Decentralized Identity Verification (DID)
+description: A decentralized identity verification on the blockchain.
```

Comment:
Your description doesn't really add any additional information that wasn't in your title.